app/services/classification/base.rb

```diff
@@ -178,11 +179,17 @@ def update_generic_matcher(lookup_values, options)
       computed_lookup_value
     end
 
-    def update_array_matcher(should_avoid_duplicates, lookup_values, options)
+    def update_array_matcher(default, should_avoid_duplicates, lookup_values, options)
       elements = []
       values = []
       element_names = []
 
+      unless default.nil?
+        values += default
+        elements << _('default_value')
```

Comment:
This is shown in the "where did this value come from?" help text in the UI, isn't it?  If so, don't use an underscore as it's a user-visible string.  Either "default value" or perhaps use the model attribute context-sensitive style, `s_("LookupKey|Default value")`